docs/source/examples.rst

```diff
@@ -216,7 +216,7 @@ Animations
 
 .. manim:: MovingAngle
     :ref_classes: Angle
-    :ref_functions: Mobject.rotate
+    :ref_methods: Mobject.rotate
```

Comment:
Still not working:
https://manimce--1209.org.readthedocs.build/en/1209/examples.html#movingangle

Reply:
Wait, is it because you wrote Mobject vs. MObject?

Reply:
@behackl : do you have an idea?

Reply:
`Mobject.rotate` has no docstring. There is nothing this could link to: https://manimce--1209.org.readthedocs.build/en/1209/reference/manim.mobject.mobject.Mobject.html#manim.mobject.mobject.Mobject

Reply:
Ohh, I am so dumb, I just wrote the docstring, but on another pr:
https://github.com/ManimCommunity/manim/pull/1206/files
Sorry for causing this confusion.

Reply:
Ah... in which case it might be better to review/change the instances of :ref_functions: to :ref_methods: after #1206 has been merged.

Reply:
Okay, it is now working, see e.g. this example: https://manimce--1209.org.readthedocs.build/en/1209/examples.html#rotationupdater
I suggest we merge this pr now and rewrite the other examples in a follow-up pr after #1206 is merged.